source/infrastructure/pipeline/e2e-tests/basic-ec2-start-stop.test.ts

```diff
@@ -0,0 +1,65 @@
+/*****************************************************************************
+ *  Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.   *
+ *                                                                            *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may   *
+ *  not use this file except in compliance with the License. A copy of the    *
+ *  License is located at                                                     *
+ *                                                                            *
+ *      http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                            *
+ *  or in the 'license' file accompanying this file. This file is distributed *
+ *  on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,        *
+ *  express or implied. See the License for the specific language governing   *
+ *  permissions and limitations under the License.                            *
+ *****************************************************************************/
+import * as ec2 from "@aws-sdk/client-ec2";
+import * as dynamodb from "@aws-sdk/client-dynamodb";
+
+import {resourceParams} from "./basic-ec2-start-stop.test.resources";
+import {delayMinutes} from "./index";
+import {getInstanceState} from "./utils/ec2-test-utils";
+import {createSchedule, currentTimePlus} from "./utils/schedule-test-utils";
+
+
+const ec2Client = new ec2.EC2Client({});
+const dynamoClient = new dynamodb.DynamoDBClient({});
```

Comment:
Should we be using our solution custom user agent here? I'm a little torn because it's not really necessary in test code, but on the other hand if customers are running this pipeline we would want those metrics to show up.

Reply:
This would be a good point of discussion to have with the team.